openfeature-provider/src/main/java/com/spotify/confidence/ConfidenceFeatureProvider.java

```diff
@@ -32,58 +30,26 @@ public class ConfidenceFeatureProvider implements FeatureProvider {
   /**
    * ConfidenceFeatureProvider constructor
    *
-   * @param confidence an instance of the Confidence
+   * @param confidenceWrapper obtained via {@link #createConfidenceWrapper(Confidence.Builder)}
    */
-  public ConfidenceFeatureProvider(Confidence confidence) {
-    this.confidence = confidence;
+  public ConfidenceFeatureProvider(ConfidenceWrapper confidenceWrapper) {
```

Comment:
Did you consider having the constructor take a `Confidence.Builder` and mutate that builder there, instead of having the `ConfidenceWrapper`?